Sources/MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -4,65 +4,85 @@ import MapboxCoreNavigation
 import MapboxDirections
 
 /**
- `InstructionsBannerViewDelegate` provides methods for reacting to user interactions in `InstructionsBannerView`.
- */
-public protocol InstructionsBannerViewDelegate: VisualInstructionDelegate {
-    /**
-     Called when the user taps the `InstructionsBannerView`.
-     */
-    func didTapInstructionsBanner(_ sender: BaseInstructionsBannerView)
-    
-    /**
-     Called when the user swipes either left, right, or down on the `InstructionsBannerView`
-     */
-    func didSwipeInstructionsBanner(_ sender: BaseInstructionsBannerView, swipeDirection direction: UISwipeGestureRecognizer.Direction)
-}
-
-public extension InstructionsBannerViewDelegate {
-    /**
-     `UnimplementedLogging` prints a warning to standard output the first time this method is called.
-     */
-    func didTapInstructionsBanner(_ sender: BaseInstructionsBannerView) {
-        logUnimplemented(protocolType: InstructionsBannerViewDelegate.self, level: .debug)
-    }
+ A banner view that contains the current step instruction and responds to tap and swipe gestures.
+ 
+ This class responds and gets updated as the user progresses along a route according to the `NavigationComponent` and `BaseInstructionsBannerView` protocol.
+*/
+@IBDesignable
+open class InstructionsBannerView: BaseInstructionsBannerView, NavigationComponent {
     
     /**
-     `UnimplementedLogging` prints a warning to standard output the first time this method is called.
+     Updates the instructions banner info as the user progresses along a route.
+     
+     - parameter service: The `NavigationService` instance that passes the instruction.
+     - parameter instruction: The `VisualInstructionBanner` instance to be presented.
+     - parameter routeProgress: The`RouteProgress` instance that the instruction banner view is updating.
      */
-    func didSwipeInstructionsBanner(_ sender: BaseInstructionsBannerView, swipeDirection direction: UISwipeGestureRecognizer.Direction) {
-        logUnimplemented(protocolType: InstructionsBannerViewDelegate.self, level: .debug)
-    }
-}
-
-/// :nodoc:
-@IBDesignable
-open class InstructionsBannerView: BaseInstructionsBannerView, NavigationComponent {
     public func navigationService(_ service: NavigationService, didPassVisualInstructionPoint instruction: VisualInstructionBanner, routeProgress: RouteProgress) {
         update(for: instruction)
     }
 }
 
-/// :nodoc:
+/**
+ A banner view that contains the current step instruction along a route and responds to tap and swipe gestures, as the base of `InstructionsCardView` and `InstructionsBannerView`.
+*/
 open class BaseInstructionsBannerView: UIControl {
+    
+    /**
+     A view that contains a simple image indicating a type of maneuver.
```

Comment:
```suggestion
     A view that contains an image indicating a type of maneuver.
```

I think it'd be better to have a neutral phrasing. Nit: maybe it'd be worth noting what component will generate these images.